compiler/AST/AstToText.cpp

```diff
@@ -1135,12 +1163,11 @@ void AstToText::appendExpr(CallExpr* expr, bool printingType)
         mText += name->unresolved;
       }
 
-      // NOAKES 2015/02/09 Treating all calls with 2 actuals as binary operators
-      // Lydia 2015/02/17 ... except homogeneous tuple inner workings.
-      else if (expr->numActuals() == 2)
+      // Format binary operators in infix notation
+      else if (expr->numActuals() == 2 && looksLikeInfixOperator(fnName))
       {
-        UnresolvedSymExpr* name     = toUnresolvedSymExpr(expr->baseExpr);
-        if (printingType && strcmp(name->unresolved, "*") == 0)
+        // Lydia 2015/02/17 ... except homogeneous tuple inner workings.
```

Comment:
I think it's fair to remove my name and that date from the comment and just let it talk about homogeneous tuples on their own

Reply:
I'll update the PR message and that comment, and add a test of the `var a = something;` forms.  I'll try to get to it tonight.

Reply:
I did those things, and also added a third variant of the test where the things being set to different expressions are record members.

Squashed again.

I also noticed that a few of the function arg tests I'd written as `proc foo(args);` instead of `proc foo(args) {}`; I fixed those.

I also noticed that in the variants where `something = x.type`, chpldoc emits an extra space after the `x.type` in its output, visible easily in the function arg .good file since there's a `)` following, and less easily in an editor in the other two.  E.g.,

```
.. function:: proc tuple3Default(arg = (x, y, z))
.. function:: proc dotTypeDefault(arg = x.type )
```

I can file a new issue and future for that, but probably not tonight.